### DISCUSSION TITLE
The medium of the "Vacuum" is electromagnetic radiation

### DISCUSSION THREAD

**Discussion by budrap00** [a-cosmology-group/acg#285] [Astrophysics and Physics]
From a [Quora answer](https://qr.ae/p2V2cf):

![image](https://github.com/user-attachments/assets/0bb1c4ae-5d3e-4a02-af06-c8f821859c49)

**Comment by HanDeBruijn** [comment]
A decent reference might be Wikipedia's [Speed of sound](https://en.wikipedia.org/wiki/Speed_of_sound).
The formula that comes most nearby for the speed of a sound **wave** is

$$
c = \sqrt{\gamma\frac{p}{\rho}} \quad \mbox{equivalently} \quad V = \sqrt{(\gamma P/\rho)}
$$

where $\gamma$ is the [Heat capacity ratio](https://en.wikipedia.org/wiki/Heat_capacity_ratio). Theoretical values are around $\gamma=1.4$ . So I don't understand where a value of $\gamma=2$ comes from in the first place. And of course, when considering values for pressure and density in a vacuum / aether : how are these **observed / measured** ?

>Nice demonstration that the medium of the “Vacuum” is electromagnetic radiation.

???

**Reply by budrap00** [reply]
Yes, what the pressure term means or where it comes from is not clear. It might just be a model fitting exercise. However the medium of the vacuum is observed to be electromagnetic radiation, so...

**Reply by LifeIsStrange** [reply]
off topic but recently saw this paper mentioning that the behaviour of light during speed of sound barrier must violate the mass equivalence relation e=mc2
https://elpub.bib.uni-wuppertal.de/servlets/MCRFileNodeServlet/duepublico_derivate_00000942/88-ESound%20BarrierA.pdf

**Comment by genotics** [comment]
Han is spot on with his reference. I would only suggest that you take a look at the Prandtl-Glauert transformation (see Wikipedia) which relies on the Prandtl-Glauert factor.  Basic laws of electromagnetism have analogous laws in hydrodynamics, e.g., Biot-Savart. See <https://archive.org/details/fundamentalsofhy0000tiet/page/206/mode/2up?q=biot&view=theater>

 Some have concluded that electromagnetism is a gas phenomenon, and that particles are vortex structures.

**Reply by LifeIsStrange** [reply]
related about quantum vortexes
https://www.quantamagazine.org/physicists-spot-quantum-tornadoes-twirling-in-a-supersolid-20241106/

**Comment by LifeIsStrange** [comment]
My comment is off topic, but since you guys are mentioning electromagnetism, I recently discovered magnetic fields can be twisted:
https://en.wikipedia.org/wiki/Magnetic_helicity

BTW 
> The radial current in the circuit is on the order of 3×10^9 [amperes](https://en.wikipedia.org/wiki/Ampere).[[2]](https://en.wikipedia.org/wiki/Heliospheric_current_sheet#cite_note-israelevich2001-2)

Would it be feasable to extract "unlimited" free energy by plugging a cable to the nearby space electric field?
https://en.wikipedia.org/wiki/Heliospheric_current_sheet

**Comment by genotics** [comment]
If supersolid, then superfluid too.
